### ISSUE TITLE
Should list PNI be replaced by ONIX list 44.

### ISSUE THREAD

**Issue by mdovey** [bic-org-uk/bic-lcf#319]
The PNI codelist is currently based on ONIX List 44 (i.e. is subset set).

For other ONIX lists 5 (Manifestation ID Type), 15 (Title Type) ,17 (Contributor Role), 19 (Contributor Type), we have moved to importing the list wholesale from the ONIX schema (hence include the entire list).

Is there any reason not to also incorporate ONIX Code List 44 wholesale?

Are there any other code lists which are partial ONIX code lists which we should incorporate wholesale?

**Comment by franciscave** [comment]
We have also moved to use ONIX code list 11 in place of our code list UNC.

However, in the case of ONIX code list 44, I'm not sure that wholesale use of this list is a good idea. Only a small subset of the code values apply to individual persons, and those are the ones that we currently support. Do we really want to make it allowable, for example, for ONIX code values such as '23' (VAT registration number) to be used as a scheme for identifying library patrons? Maybe the answer is that, since the schemes used is a matter for the library in question, and they might allow organisations as well as individual persons to be registered as library patrons, we should not constrain the choices that libraries make. In which case we can remove PNI in favour of allowing the whole of ONIX code list 44.

**Comment by mdovey** [comment]
I think that might be a typo in the documentation. In the schema UNC maps to ONIX code list 19 (unamed persons), and there doesn't appear to be an ONIX Code List 11 in the current ONIX issue (and the only Code List 11 I can find from some years ago is for e-publisher format code).

**Comment by franciscave** [comment]
You're right. The documentation should refer to ONIX code list 19, not 11.

**Comment by mdovey** [comment]
I think there is an argument that a library might have legal persons as patrons rather than natural persons, in which case other entities in ONIX list 44 might apply.

I suggest leaving as is for 1.3.0 (given we have no concrete requests) but start a discussion on this at the technical panel after the release of 1.3.0.
